src/Components/Products/Products.js

```diff
@@ -4,27 +4,27 @@ import "./style.css";
 export default class Products extends Component {
```

Comment:
This component is unclear. Is it getting a list of products? Is it getting a single product? What is the purpose of this component? Is it clear from the your variable/function definitions.